deepinfra/clients/deepinfra.py

```diff
@@ -40,18 +36,9 @@ def post(self, data, config=None):
             "User-Agent": USER_AGENT,
             "Authorization": f"Bearer {self.auth_token}",
         }
-        for attempt in range(self.max_retries + 1):
```

Comment:
So you dropped the retries logic, which is probably OK, as long as we replace it with something equivalent (very likely requests has some builtin, you can dig a bit). Otherwise what is the purpose?

Reply:
The idea is to leave this decision to the user.

Reply:
Guess you're right. This change expands scope of this PR which is irrelevant.